config/initializers/inflections.rb

```diff
@@ -4,12 +4,11 @@
 # Add new inflection rules using the following format. Inflections
 # are locale specific, and you may define rules for as many different
 # locales as you wish. All of these examples are active by default:
-# ActiveSupport::Inflector.inflections(:en) do |inflect|
-#   inflect.plural /^(ox)$/i, '\1en'
-#   inflect.singular /^(ox)en/i, '\1'
-#   inflect.irregular 'person', 'people'
-#   inflect.uncountable %w( fish sheep )
-# end
+ActiveSupport::Inflector.inflections(:en) do |inflect|
```

Comment:
This is the standard way to handle acronyms in Rails. But it is a global change that will affect the entire service, could this cause any issues?

From https://api.publish-teacher-training-courses.service.gov.uk/api-reference.html#schema-coursefilter

```
Possible values:

qts
pgce_with_qts
pgde
pgce
pgde_with_qts
```